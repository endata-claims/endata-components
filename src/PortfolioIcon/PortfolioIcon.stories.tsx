import React from 'react'

import PortfolioIcon from '.'

export default {
  title: 'Portfolio Icon'
}

export const Default = () => {
  return (
    <>
      <PortfolioIcon portfolio='Building' />
      <PortfolioIcon portfolio='Contents' />
      <PortfolioIcon portfolio='Restoration' />
    </>
  )
}