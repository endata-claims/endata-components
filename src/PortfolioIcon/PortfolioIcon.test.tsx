import React from 'react'
import { shallow } from 'enzyme'

import PortfolioIcon from '.'

describe('Paper tests', () => {
  const renderWrapper = (props: any) => shallow(<PortfolioIcon {...props} />)

  describe('snapshots', () => {
    it('should render icon', () => {
      expect(renderWrapper({ portfolio: 'Building' })).toMatchSnapshot()
      expect(renderWrapper({ portfolio: 'Contents' })).toMatchSnapshot()
      expect(renderWrapper({ portfolio: 'Restoration' })).toMatchSnapshot()
    })
  })
})