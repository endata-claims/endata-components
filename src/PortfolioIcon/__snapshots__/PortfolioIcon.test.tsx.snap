// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper tests snapshots should render icon 1`] = `
<HomeIcon
  style={
    Object {
      "color": "#0f5b78",
    }
  }
/>
`;

exports[`Paper tests snapshots should render icon 2`] = `
<LiveTvIcon
  style={
    Object {
      "color": "green",
    }
  }
/>
`;

exports[`Paper tests snapshots should render icon 3`] = `
<HealingIcon
  style={
    Object {
      "color": "#d0a11e",
    }
  }
/>
`;
